pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/util/NotificationUtilTest.java

```diff
@@ -0,0 +1,21 @@
+package org.broadinstitute.dsm.util;
+
+import static org.junit.Assert.fail;
+
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+import org.junit.Test;
+
+public class NotificationUtilTest {
+
+    @Test
+    public void testStartup() {
+        Config cfg = ConfigFactory.parseResources("NotificationUtilTest.conf");
+        try {
+            var notificationUtil = new NotificationUtil(cfg);
```

Comment:
FWIW, the DSM backend code uses `var` sparingly. I use it occasionally, and only in `for` loops.
You may want to bring up usage of `var` in Process and Standards, since in the case of DSM backend, it is currently non-standard. (And Java programmers may have strong opinions about its use.)

Reply:
Added to the agenda, will leave this for now and see where the discussion goes next week.